VK_V1.0/VK_V1.0/Model/News/View.swift

```diff
@@ -0,0 +1,10 @@
+// View.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Кол-во просмотров
+struct View: Decodable {
```

Comment:
```suggestion
struct Views: Decodable {
```
та же история, относится ко всем просмотрам, а не к какому то конкретному